pkg/operator/ceph/nfs/nfs.go

```diff
@@ -124,6 +124,9 @@ func (r *ReconcileCephNFS) upCephNFS(n *cephv1.CephNFS) error {
 			return errors.Wrap(err, "failed to create ceph nfs service")
 		}
 
+		// with upgrades remove old server from database
+		r.removeServerFromDatabase(n, getDeprecatedNFSNodeID(n, id))
```

Comment:
Does this method log an error if the id was not in the database? Or is it a no-op if not in the database?

Reply:
yes